aggregations/db_tables/daily_ingoing_transactions_per_account_count.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+
+from . import DAY_LEN_SECONDS, daily_start_of_range
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyIngoingTransactionsPerAccountCount(PeriodicAggregations):
+    @property
+    def sql_create_table(self):
+        # Suppose we have at most 10^5 (100K) transactions per second.
+        # In the worst case, they are all from one account.
+        # It gives ~10^10 transactions per day.
+        # It means we fit into BIGINT (10^18)
+        return '''
+            CREATE TABLE IF NOT EXISTS daily_ingoing_transactions_per_account_count
+            (
+                collected_for_day          DATE   NOT NULL,
+                account_id                 TEXT   NOT NULL,
+                ingoing_transactions_count BIGINT NOT NULL,
+                CONSTRAINT daily_ingoing_transactions_per_account_count_pk PRIMARY KEY (collected_for_day, account_id)
+            );
+            CREATE INDEX IF NOT EXISTS daily_ingoing_transactions_per_account_count_idx
+                ON daily_ingoing_transactions_per_account_count (collected_for_day, ingoing_transactions_count DESC)
+        '''
+
+    @property
+    def sql_drop_table(self):
+        return '''
+            DROP TABLE IF EXISTS daily_ingoing_transactions_per_account_count
+        '''
+
+    @property
+    def sql_select(self):
+        # Ingoing transactions for user X aren't only transactions where receiver_account_id == X.
+        # We need to find all chains with receipts where X was the receiver.
+        # It's important to add 10 minutes to receipt border, we should pack the transaction
+        # with all their receipts together, or the numbers will not be accurate.
+        # Other receipts from the next day will be naturally ignored.
+        # Transactions border remains the same, taking only transactions for the specified day.
+        # If you want to change 10 minutes constant, fix it also in PeriodicAggregations.is_indexer_ready
+        return '''
+            SELECT
+                receipts.receiver_account_id,
+                COUNT(DISTINCT transactions.transaction_hash) AS transactions_count
+            FROM transactions
+            LEFT JOIN receipts ON receipts.originated_from_transaction_hash = transactions.transaction_hash
+                AND transactions.block_timestamp >= %(from_timestamp)s
+                AND transactions.block_timestamp < %(to_timestamp)s
+            WHERE receipts.included_in_block_timestamp >= %(from_timestamp)s
+                AND receipts.included_in_block_timestamp < (%(to_timestamp)s + 600000000000)
```

Comment:
This constraint looks like optimization rather than necessity. Is it really necessary? (I am asking because on mainnet we had an incident that caused some receipts landed half a year later...)

Reply:
It's about optimization
I was playing with query plans for 2 hours 🤯 
It's running 70-150s if I write it straight-forward
But I will re-check what it would be if I remove this exact condition

Reply:
Let's mention these valuable insights in the comments to the code, so whoever needs to do anything with it in the future can see that quite some time has already been put into it exploring different approaches :+1:

Reply:
> on mainnet we had an incident that caused some receipts landed half a year later

It causes the problem that we can't trust previously computed data, or we should re-compute all the data every night 🙁 
I suggest not to take such cases into account